index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Tribute Page</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <div id="main">
+      <h1 id= "title">Dr. Norman Borlaug</h1>
+      <p>The man who saved a billion lives</p>
```

Comment:
Nice job on making this suggestion, but I have some general recommendations to make your review comments even better:
- Could you kindly take a quick look and make any necessary edits to improve the clarity of your comment? I noticed some small grammar and spelling errors, and I think it would make a big difference for the student if we could correct them. 💯

- It would be helpful if you could provide more context or explain **why** you are suggesting the use of the `p` tag instead of the `h3` tag.

- Kindly try to add the **Add code** GitHub comment feature for the `p` and `h3` tags to make it more readable. What do you think? 🙋‍♂️